NETDefault/Gateway.Buckaroo/BuckarooConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Gateway.Buckaroo;
+
+public class BuckarooConfiguration
+{
+    public const string SectionName = nameof(BuckarooConfiguration);

```

Comment:
Should we just remove this `SectionName` and use `nameof()` as convention for the name of the section?

Reply:
Ah! Actually better yeah!